scripts/run_integration_tests.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Usage - run commands from repo root:
+# To check if new changes to the library cause changes to any snapshots:
+#   ./scripts/run_integration_tests.sh
+# To regenerate snapshots:
+#   UPDATE_SNAPSHOTS=true ./scripts/run_integration_tests.sh
+
+set -e
+
+# To add new tests create a new ts file in the 'integration_tests/lambda' directory, append it as a js file to the STACK_CONFIGS array
+# as well as creating a name for the snapshots that will be compared in your test. Add those snapshot names to the TEST_SNAPSHOTS and CORRECT_SNAPSHOTS arrays.
+# Note: Each yml config, test, and correct snapshot file should be at the same index in their own array. e.g. All the files for the forwarder test are at index 0.
+#       In order for this script to work correctly these arrays should have the same amount of elements.
+STACK_CONFIGS=("lambda-stack.js" "lambda-nodejs-function-stack.js")
+TEST_SNAPSHOTS=("test_lambda_stack_snapshot.json" "test_lambda_nodejs_function_stack_snapshot.json")
+CORRECT_SNAPSHOTS=("correct_lambda_stack_snapshot.json" "correct_lambda_nodejs_function_stack_snapshot.json")
```

Comment:
I wonder if there is a way to refactor this code so that when we add a new stack, we only have to add it once, and the file names are automatically generated?

Reply:
What if we added two directories `correct_snapshots` and `test_snapshots` and used the for loop to iterate over all three directories (including `stacks`)? Since the files are already sorted by name, they should be run in the correct order.

Reply:
I think you should go with whatever you think the best developer experience will be. You could also have the list of stacks in this file, but then use the stack names to know the names of the test snapshot and correct snapshot files (because they all follow the same pattern).

Reply:
Thanks for the feedback! I went with the latter option and just used the stack names to generate those snapshots instead of having to list them. Now that I think about it, I'm not sure that a list for both was ever required!